apps/music/index.html

```diff
@@ -63,28 +62,72 @@ <h1 id="title-text"><bdi data-l10n-id="music"></bdi></h1>
     </gaia-header>
     <div id="views">
       <div id="views-tiles" class="view bb-tabpanel hidden" aria-labelledby="tabs-mix" role="tabpanel">
-        <form id="views-tiles-search" role="search" class="skin-dark">
-          <button id="views-tiles-search-close" data-l10n-id="search-close" type="submit">close</button>
-          <p>
-            <input id="views-tiles-search-input" type="search" dir="auto"
-                   x-inputmode="verbatim"
-                   data-l10n-id="search-music" placeholder="SearcH MusiC">
-            <button id="views-tiles-search-clear" type="reset">Clear</button>
-          </p>
-        </form>
-        <div id="views-tiles-anchor"></div>
+        <div id="views-scrollable-tiles-anchor">
+          <form id="views-tiles-search" role="search" class="skin-dark">
+            <button id="views-tiles-search-close" data-l10n-id="search-close" type="submit">close</button>
+            <p>
+              <input id="views-tiles-search-input" type="search" dir="auto"
+                     x-inputmode="verbatim"
+                     data-l10n-id="search-music" placeholder="SearcH MusiC">
+              <button id="views-tiles-search-clear" type="reset">Clear</button>
+            </p>
+          </form>
+          <div id="views-tiles-anchor"></div>
+        </div>
+        <!-- BB tabs -->
+        <div class="tabs">
+          <ul role="tablist" class="bb-tablist skin-dark">
+            <li role="presentation">
+              <a data-option="mix" role="tab" aria-selected="false" aria-controls="views-tiles" data-icon="grid" data-l10n-id="mix-tab"></a>
```

Comment:
Nit: I think it would be nice if we wrapped this to 80 columns. That way, no one needs to scroll horizontally when looking at the HTML. (I know the original tabs HTML wasn't wrapped either, but now's a good time to do it.)